contracts/utils/cryptography/MerkleProof.sol

```diff
@@ -193,6 +192,10 @@ library MerkleProof {
      * CAUTION: Not all Merkle trees admit multiproofs. To use multiproofs, it is sufficient to ensure that: 1) the tree
      * is complete (but not necessarily perfect), 2) the leaves to be proven are in the opposite order they are in the
      * tree (i.e., as seen from right to left starting at the deepest layer and continuing at the next layer).
+     *
+     * NOTE: The _empty set_ (i.e. the case where `proof.length == 1 && leaves.length == 0`) is considered a noop,
```

Comment:
```suggestion
     * NOTE: The _empty set_ (i.e. the case where `proof.length == 1 && leaves.length == 0`) is considered a no-op,
```

Reply:
I think the correct way to write this is NOP, no-op or NOOP (see [here](https://en.wikipedia.org/wiki/NOP_(code))). Feel free to ignore this nit comment.